### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by jmini**
I have done some modifications of the readme file.

Screenshots
--------------
I have added 2 screenshots because I think it will help the users to understand what they will get.

**Important**: It is not necessary to include those files in the xpi archive. Is it possible to exclude them from the build?

GitHub Download link
--------------
`latest-release` was outdated (pointing to the `asciidoctorjslivepreview.xpi` file of version `v0.2.0`)

I think it is more maintainable to use a link to the latest release page (redirection is handled by GitHub) as I proposed it. For the users it adds a click (on the green download button) on the linked page, but I think it is OK. Better 2 clicks than an outdated install.

Text modifications
--------------
I have done some small texts modifications.

Feel free to give me feedback.

### CODE REVIEW

**Comment by Mogztter** [comment]
Wow that's better :+1: 

> It is not necessary to include those files in the xpi archive. Is it possible to exclude them from the build?

I don't know, I will read the documentation to find out.

> I think it is more maintainable to use a link to the latest release page (redirection is handled by GitHub) as I proposed it. For the users it adds a click (on the green download button) on the linked page, but I think it is OK. Better 2 clicks than an outdated install.

Absolutely!

File: README.adoc [hunk 1]
```diff
@@ -3,18 +3,26 @@ Guillaume Grossetie
 :sources: https://github.com/asciidoctor/asciidoctor-firefox-addon
 :license: https://github.com/asciidoctor/asciidoctor-firefox-addon/blob/master/LICENSE
 :ff-addon-sdk-install: https://addons.mozilla.org/en-US/developers/docs/sdk/latest/dev-guide/tutorials/installation.html
-:ff-addon-site: https://addons.mozilla.org/
-:latest-release: https://github.com/asciidoctor/asciidoctor-firefox-addon/releases/download/v0.2.0/asciidoctorjslivepreview.xpi
+:ff-addon-site: https://addons.mozilla.org/de/firefox/addon/asciidoctorjs-live-preview/
```
Comment (Mogztter) [inline README.adoc]: You need to remove `/de/` in the URL

**Comment by jmini** [comment]
`de/` removed in `ff-addon-site`.

**Comment by Mogztter** [comment]
Perfect, thanks!

**Pull request merged**